pepper-apis/src/main/java/org/broadinstitute/ddp/service/AgeUpService.java

```diff
@@ -106,8 +106,8 @@ public int runAgeUpCheck(Handle handle, PexInterpreter interpreter, GovernancePo
                                 candidate.getParticipantUserId(),
                                 candidate.getParticipantUserId(),
                                 candidate.getParticipantUserGuid(),
-                                policy.getStudyId(),
-                                EventTriggerType.CONSENT_SUSPENDED);
+                                candidate.getParticipantUserGuid(),
```

Comment:
This service is ran in the background and doesn't have an operator.

Reply:
Done